media-sound/drumkv1/drumkv1-0.8.2.ebuild

```diff
@@ -1 +0,0 @@
-drumkv1-9999.ebuild
```

Comment:
While you're here, mind fixing the copyright year? Seems we missed it last time.